src/controller/CommissioningDelegate.h

```diff
@@ -441,6 +441,7 @@ struct GeneralCommissioningInfo
 {
     uint64_t breadcrumb          = 0;
     uint16_t recommendedFailsafe = 0;
+    //TODO should I be adding something in here?
```

Comment:
These TODOs should probably expanded like

`TODO: determine if usage of maxCummulativeSeconds is required here after being added in #19582.`